browser/brave_ads/tabs/ads_tab_helper.cc

```diff
@@ -55,14 +55,10 @@ AdsTabHelper::AdsTabHelper(content::WebContents* web_contents)
 #if !BUILDFLAG(IS_ANDROID)
   // See "background_helper_android.h" for Android.
   BrowserList::AddObserver(this);
-
-  OnBrowserSetLastActive(BrowserList::GetInstance()->GetLastActive());
-#else
-  // Set `is_browser_active_` to `true` because `BrowserList` class is not
-  // available on Android.
-  is_browser_active_ = true;
 #endif  // !BUILDFLAG(IS_ANDROID)
 
+  MaybeSetBrowserIsActive();
```

Comment:
Call this for both desktop and Android so that `MaybeNotifyBrowserDidBecomeActive();` is called.

Reply:
The bug was that `MaybeNotifyBrowserDidBecomeActive();` was not called for Android.